otsdaq-mu2e-tracker/Generators/TrackerVST001_generator.cc

```diff
@@ -0,0 +1,969 @@
+///////////////////////////////////////////////////////////////////////////////
+// P.Murat: test version, implements multiple DTC read tests
```

Comment:
I'd like to keep the actual mechanics of readout centralized in artdaq_mu2e, if possible. Can we look at this code and use Mu2eEventReceiverBase for the common parts?